sysfs/system_cpu.go

```diff
@@ -76,12 +77,19 @@ func (fs FS) NewSystemCpufreq() ([]SystemCPUCpufreqStats, error) {
 			return []SystemCPUCpufreqStats{}, err
 		}
 
-		cpufreq, err = parseCpufreqCpuinfo(cpuCpufreqPath)
-		if err != nil {
-			return []SystemCPUCpufreqStats{}, err
-		}
-		cpufreq.Name = cpuNum
-		systemCpufreq = append(systemCpufreq, *cpufreq)
+		i := i // https://golang.org/doc/faq#closures_and_goroutines
+		g.Go(func() error {
```

Comment:
Could you add a comment somewhere to explain why we are doing this in parallel, because without the knowledge that the kernel intentionally puts in delays it seems like this wouldn't be needed.

Reply:
Sure.